src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -263,7 +263,9 @@ function assertValidProps(component, props) {
   }
   invariant(
     props.style == null || typeof props.style === 'object',
-    'The `style` prop expects a mapping from style properties to values, ' +
+    'Please check the render method of ' +
+     component._currentElement._owner._instance.__proto__.constructor.displayName +
```

Comment:
You need to have an if-check, because there is no guarantee that _owner isn't null (element could have been created outside of a render method).